book/src/SUMMARY.md

```diff
@@ -46,4 +46,6 @@
     - [Unit Testing](unit-testing.md)
     - [Functional Testing](functional-testing.md)
 - [Drivers](drivers.md)
+- [Reference]()
```

Comment:
This should not be empty in ()?

Reply:
mdbook currently doesn't allow non-link summary item. See https://github.com/rust-lang/mdBook/issues/483. I can add a new reference.md, if desired.